src/components/Home/Categories.tsx

```diff
@@ -13,18 +16,25 @@ export default function Categories({
   useEffect(() => {
     const loadPopularCategories = async () => {
       const data = await getPopularTags(size)
-
       setCategories(data.aggregations?.popular_tags?.buckets)
     }
     loadPopularCategories()
   }, [size])
 
   return (
-    <div className={styles.container}>
-      {categories?.map((category, i) => {
-        console.log(category)
-        return <div key={i}>{category.key}</div>
-      })}
-    </div>
+    <HomeSection title="Popular Categories">
+      <div className={styles.categories}>
+        {categories?.map((category, i) => (
+          <Link
+            key={i}
+            href={{ pathname: '/search', query: { tag: category.key } }}
+          >
+            <a>
+              <Box className={styles.box}>{category.key}</Box>
+            </a>
+          </Link>
+        ))}
```

Comment:
suggestion(non-blocking): it would be nice to have some fallback content in the section if for whatever reason categories could not be loaded. Just a simple text like `Categories could not be loaded.` or similar.